eng/Versions.props

```diff
@@ -226,11 +226,11 @@
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension60x64Version>$(MicrosoftAspNetCoreAzureAppServicesSiteExtension60Version)</MicrosoftAspNetCoreAzureAppServicesSiteExtension60x64Version>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension60x86Version>$(MicrosoftAspNetCoreAzureAppServicesSiteExtension60Version)</MicrosoftAspNetCoreAzureAppServicesSiteExtension60x86Version>
     <!-- 3rd party dependencies -->
-    <AngleSharpVersion>0.9.9</AngleSharpVersion>
+    <AngleSharpVersion>0.10.0</AngleSharpVersion>
```

Comment:
Changes to this file are to prevent `System.Net.Http 4.1.0` from being brought in transitively